ログアウト画面のcssを作成

koron56 opened the pull request:
ログアウト画面のcssの作成とそれに伴いlogout.phpの変更

koron56 committed:
ログアウト画面のcssを作成

ログアウト画面のcssの作成とそれに伴いlogout.phpの変更

reiya97 reviewed (changes requested):
ログアウト画面はホームにだけ行けるようにしてくれたらいいよ．あとpタグは消しちゃってh2タグだけでOK

koron56 committed:
ログアウト画面とcssの変更

32行目のpタグの消去
33行目のaタグのhrefをlogin_form.phpから./index.phpに変更と文字をマイページへからホームへに変更
h2タグの色をオレンジから黒色に変更

reiya97 reviewed (approved):
確認しました！
